app/components/meetings/api.js

```diff
@@ -272,6 +272,45 @@ export default class Api
     await axios.put(url, file, config);
   }
 
+  async getMeetingDocuments(options) {
+    const documents = await this.http.get(`/api/v2016/documents`, options).then(res => res.data).catch(tryCastToApiError);
+    return documents;
+  } 
```

Comment:
passing options directo to http library is bad. shoudl rename and change siganture
eg: 
```suggestion
  async queryMeetingDocuments({cache, ...params} ) {
  // params: { q, f, l, sk, fo } 
  
  cache = !!cache; // force boolean
  
    const documents = await this.http.get(`/api/v2016/documents`, { params, cache }).then(res => res.data).catch(tryCastToApiError);
    return documents;
  } 
```

Reply:
The issue was fixed.